### PR TITLE
Cortex XSOAR replace name

### PR DESCRIPTION
**Pull request by dansterenson**
Replaced Cortex XSOAR in the upload artifacts step.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: .gitlab/ci/on-push.yml [hunk 1]
```diff
@@ -136,6 +136,15 @@ prepare-testing-bucket-mpv2:
     - demisto-sdk find-dependencies -idp Tests/id_set.json --all-packs-dependencies -o $ARTIFACTS_FOLDER/packs_dependencies.json
     - section_end "Calculate Packs Dependencies"
 
+    - section_start "Replace Cortex XSOAR" --collapsed
+    - find $ARTIFACTS_FOLDER/content_packs -type f -exec sed -i -e 's/Cortex XSOAR/'"$PRODUCT_NAME"'/gI' {} \;
+    - zip -r $ARTIFACTS_FOLDER/content_packs.zip $ARTIFACTS_FOLDER/content_packs
+    - zip -r $ARTIFACTS_FOLDER/all_content.zip $ARTIFACTS_FOLDER/all_content
+    - zip -r $ARTIFACTS_FOLDER/content_new.zip $ARTIFACTS_FOLDER/content_new
+    - zip -r $ARTIFACTS_FOLDER/content_test.zip $ARTIFACTS_FOLDER/content_test
```
Comment (yaakovi) [inline .gitlab/ci/on-push.yml:144]: make sure files are added at the root of the file

**Review by yaakovi — APPROVED** [review]
please sort the folder issue within the zip and check that the bucket is valid

File: .gitlab/ci/on-push.yml [hunk 2]
```diff
@@ -138,6 +138,18 @@ prepare-testing-bucket-mpv2:
     - demisto-sdk find-dependencies -idp Tests/id_set.json --all-packs-dependencies -o $ARTIFACTS_FOLDER/packs_dependencies.json
     - section_end "Calculate Packs Dependencies"
 
+    - section_start "Replace Cortex XSOAR" --collapsed
+    - path="$PWD"
+    - cd $ARTIFACTS_FOLDER
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply .gitlab/ci/on-push.yml:143]: Done

File: .gitlab/ci/on-push.yml [hunk 3]
```diff
@@ -138,6 +138,18 @@ prepare-testing-bucket-mpv2:
...
+    - find content_packs -type f -not \( -path "*/ReleaseNotes/*" \) -exec sed -i -e 's/Cortex XSOAR/'"$PRODUCT_NAME"'/gI' {} \;
+    - zip -r content_packs.zip content_packs
+    - zip -r all_content.zip all_content
+    - zip -r content_new.zip content_new
+    - zip -r content_test.zip content_test
+    - rm -rf content_packs all_content content_new content_test
+    - cd "$path"
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply .gitlab/ci/on-push.yml:150]: Done

**Pull request merged**